stream_muxer/mplex/mplex.py

```diff
@@ -1,10 +1,10 @@
 import asyncio
```

Comment:
Do we need the old muxer folder at all? If not, let's delete.

Reply:
the old muxer folder is renamed to stream_muxer. we should also have a protocol_muxer folder in the future